specification/containerservice/resource-manager/Microsoft.ContainerService/stable/2022-06-01/scenarios/managedClusters.yaml

```diff
@@ -9,41 +9,58 @@ variables:
     type: string
   azureClientSecret:
     type: string
+  
+prepareSteps:
+  - step: ManagedClusters_CreateOrUpdate
+    exampleFile: ..\examples\ManagedClustersCreate_Update.json
+    requestUpdate:
+      - remove: /parameters/identity
+      - remove: /parameters/sku
+      - remove: /parameters/tags
+      - replace: /parameters/properties
```

Comment:
use operation based step if you need rewrite all the requests

Reply:
https://github.com/Azure/azure-rest-api-specs/blob/main/documentation/api-scenario/references/ApiScenarioDefinition.md#operation-based-step

Reply:
replaced operationId